docs/guides/unity-catalog.md

```diff
@@ -121,13 +339,24 @@ resource "databricks_schema" "things" {
 }
 
 resource "databricks_grants" "things" {
-  schema = databricks_schema.things.id
+  provider = databricks.workspace  
+  schema   = databricks_schema.things.id
   grant {
     principal  = "Data Engineers"
     privileges = ["USAGE"]
   }
 }
+```
+
+## Configure external tables and credentials
+
+To work with external tables, Unity Catalog introduces two new objects to access and work with external cloud storage:
+- **Storage credentials** represent authentication methods to access cloud storage (e.g. an IAM role for Amazon S3 or a service principal for Azure Storage). Storage credentials are access-controlled to determine which users can use the credential.
```

Comment:
`**Storage credentials**` -> link to resource docs

Reply:
done